src/topology/subset_properties.lean

```diff
@@ -627,6 +627,18 @@ iff.intro (assume h, h.image hf.continuous) $ assume h, begin
   rwa [hf.induced, nhds_induced, ←map_le_iff_le_comap]
 end
 
+/-- A closed embedding is proper, i.e., inverse images of compact sets are compact. -/
```

Comment:
This docstring is very confusing to me since, as far as I can tell, the actual conclusion of the lemma, `tendsto f (filter.cocompact α) (filter.cocompact β)` is *not* equivalent to "inverse images of compact sets are compact" for a general map between general topological spaces.

Reply:
Changed docstring to "inverse images of compact sets are contained in compacts". Hopefully that's clearer?